apps/server/swagger/data-api.yml

```diff
@@ -10,6 +10,10 @@
         schema:
           type: string
         description: ID of the category
+      - name: entityName
+        description: Filter by entity name. Separate multiple values with pipe character (|)
```

Comment:
Why `|` instead of `,`? using commas in query parameters is standard. We can also update the Lectern schema to remove some more special characters from schema names (no `,` or `|`)

Reply:
Maybe its even better to use the HTTP query param pattern where an array is created when the query parameter is used multiple times:

`GET /path/to/thing?entityName=donor&entityName=specimen&entityName=sample`

This is actually how Swagger will build arrays when you specify an array query param, and express will parse this as a single array. This also means there is no custom parsing needed to split a value into an array, which has led to comments in this review about how best to standardize that (from Justin below and me above here).

Reply:
While I realise this is just an example, there are multiple valid formats to build http array query params, without clear consensus on an approach. Enforcing one would limit third party integration, no?

On the other hand, the pipe here is meant to just be an alternative default separator, for what in my initial comment was "don't hardcode comma as the only possible option", since CSVs allow commas within a value if you wrap the value in single/double quotes.
I'm currently dealing with that in the wastewater implementation.

Reply:
This makes more sense. instead of handling special character separators ourselves will use the standard query param array format.